docs/02-admin/FAQ.md

```diff
@@ -107,6 +107,18 @@ so for a correct ready count you have to go to the queue specific overview.
 | --------------------------------------------------------- | --------------------------------------------------- | ------------------------------------------------------------------- |
 | ![Queued messages](../images/rabbit_queue_overview.png) | ![Queue overview](../images/rabbit_queue_tab.png) | ![Message Queue Overview](../images/rabbit_messages_overview.png) |
 
+## Discarding queued messages
```

Comment:
I did write an entry about the circular message thing. Can they be combined? At least they should be next to one anotherI think

Reply:
Can you point me to it? I don't see it.

Reply:
> RabbitMQ shows a really high publishing rate

Line 212